cedar-policy-validator/src/rbac.rs

```diff
@@ -1308,29 +1335,15 @@ mod test {
         let policy =
             parse_policy_template(None, "permit(principal is bar, action, resource);").unwrap();
 
-        let validate = Validator::new(schema.clone());
-        assert!(
-            validate
-                .validate_policy(&policy, ValidationMode::default())
-                .next()
-                .is_none(),
-            "Did not expect any validation notes."
-        );
+        assert_validate_policy_succeeds(schema.clone(), &policy);
```

Comment:
to avoid this `.clone()`, we could have `assert_validate_policy_succeeds()` take a `&Validator` which could be reused across multiple `assert_validate` calls